UnityProject/Assets/Scripts/Robotics/Bots/InteractableBot.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using UnityEngine;
+using Mirror;
+using Systems.MobAIs;
+
+namespace Robotics
+{
+	public class InteractableBot : NetworkBehaviour, IPredictedCheckedInteractable<HandApply>
+	{
+		HandApply interaction;
+
+		[SerializeField] private SpriteRenderer SPRITE_RENDERER;
+		[SerializeField] private Sprite EMAGGED_SPRITE;
+
+		private MobExplore mobController;
+		public MobExplore MobController
+		{
+			get
+			{
+				if (!mobController)
+				{
+					mobController = GetComponent<MobExplore>();
+				}
+				return mobController;
+			}
+		}
+		public void ClientPredictInteraction(HandApply interaction) { }
```

Comment:
if you're not going to implement prediction, use the interaction interface that has no prediction